tests/integration/component.go

```diff
@@ -583,6 +583,12 @@ func componentTests(args ...string) {
 			output := helper.CmdShouldPass("odo", append(args, "create", "nodejs:latest", componentName, "--app", appName, "--project", project, "--context", context)...)
 			Expect(output).NotTo(ContainSubstring("Warning"))
 		})
+
```

Comment:
if there is java specific code involved as well, you should change the PR description. I see the supported-java.json file being added as well. will be a good practice.

Reply:
Make sense 👍 

I could have split it into two prs but currently it is well organized so i don't want to split again. Yes, i will update the pr title.

Reply:
Done